client/js/genre.js

```diff
@@ -1,24 +1,24 @@
 /**
- * fetches youtube api calls from YoutubeServlet.java
+ * @file fetches youtube api calls from YoutubeServlet.java
  * and displays on youtube-genre.html
  */
 
-const genreBlock = document.getElementById('genres');
+document.body.onload = fetchMusicGenre();
 
 /**
- * fetches genre count hashmap from /api/youtube and updates html
+ * fetches and returns genre analysis object from /api/youtube
+ *
+ * @returns {JSON object} of youtube genreData and stats
```

Comment:
`JSON object` is not an ESLint type; if you don't want to write a specific typedef then you can just write `{object} An object containing YouTube genre data and stats`